grafana-plugin/src/models/alert_receive_channel_webhooks/alert_receive_channel_webhooks.ts

```diff
@@ -64,4 +65,33 @@ export class AlertReceiveChannelWebhooksStore {
       delete this.items[webhookId];
     });
   }
+
+  @WithGlobalNotification({
+    success: 'Webhook has been enabled.',
+    failure: 'There was an issue enabling a webhook. Please try again.',
+  })
+  async enable(integrationId: ApiSchemas['AlertReceiveChannel']['id'], webhookId: ApiSchemas['Webhook']['id']) {
+    await this._update(integrationId, { id: webhookId, is_webhook_enabled: true });
+  }
+
+  @WithGlobalNotification({
+    success: 'Webhook has been disabled.',
+    failure: 'There was an issue disabling a webhook. Please try again.',
+  })
+  async disable(integrationId: ApiSchemas['AlertReceiveChannel']['id'], webhookId: ApiSchemas['Webhook']['id']) {
+    await this._update(integrationId, { id: webhookId, is_webhook_enabled: false });
+  }
+
+  private async _update(
+    integrationId: ApiSchemas['AlertReceiveChannel']['id'],
+    webhook: Partial<ApiSchemas['Webhook']> & { id: ApiSchemas['Webhook']['id'] }
+  ) {
+    const { data } = await onCallApi().PUT('/alert_receive_channels/{id}/webhooks/{webhook_id}/', {
```

Comment:
shouldn't this have a base path instead?

Reply:
When using `onCallApi` client, path needs to be passed this way to not loose type-safety